### PR TITLE
docs: add note about user CA rotation + desktop access

### PR DESCRIPTION
**Pull request by zmb3**
In the future we'll move desktop access to a separate CA, but for
now Windows certs are generated with the user CA, so Active Directory
will need to be updated to trust the new certificate when a rotation
takes place.

### CODE REVIEW

File: docs/pages/setup/operations/ca-rotation.mdx
Comment (zmb3) [inline docs/pages/setup/operations/ca-rotation.mdx:291]: Is it poor form to have two `Adminition`s back to back like this?

File: docs/pages/setup/operations/ca-rotation.mdx
Comment (ptgott) [inline docs/pages/setup/operations/ca-rotation.mdx:293]: Rather than add the link on "exported", which could break the user's focus, I should we should add a "Read more about exporting the Teleport CA" link at the bottom of this admonition box.

File: docs/pages/setup/operations/ca-rotation.mdx
Comment (ptgott) [reply docs/pages/setup/operations/ca-rotation.mdx:291]: I've suggested moving the CA pinning warning since we use it twice in the page.

File: docs/pages/setup/operations/ca-rotation.mdx
Comment (zmb3) [reply docs/pages/setup/operations/ca-rotation.mdx:293]: I am definitely not the expert, so I defer to you on the final decision, but in this case I think the link makes sense.

As a reader, I'm thinking "you're telling me I have to do this export thing, but I have no idea what that means or how to do it. Why make me search for it?"

File: docs/pages/setup/operations/ca-rotation.mdx
Comment (zmb3) [reply docs/pages/setup/operations/ca-rotation.mdx:26]: I don't know, to be honest. This is a very old page that predates my employment at Teleport. On first glance, I don't see a problem moving this to a "further reading" section.

I guess getting such a thorough review is what I get for wrapping lines that aren't related to the actual change I'm making 😅

File: docs/pages/setup/operations/ca-rotation.mdx
Comment (zmb3) [reply docs/pages/setup/operations/ca-rotation.mdx:56]: How about "by entering the `standby` state"??

We can even include the command required to do so.

**Comment by zmb3** [comment]
@ptgott after thinking on this, many of your comments should be applied to the docs for all recent releases, but the note about desktop access + cert rotation only applies to v8 and up.

I will open a separate PR to address the comments that are not related to desktop access. Once we get that merged and backported to all appropriate branches, I'll update this PR so that it contains only the desktop access change.

Sound reasonable?

**Comment by ptgott** [comment]
@zmb3 It does!

**Comment by zmb3** [comment]
@ptgott I've applied most of your suggestions here in PR #10419.

**Comment by ptgott** [comment]
@zmb3 Thanks! So my comments here don't overlap too much with my comments in #10419, would you mind removing any changes from this PR that conflict with the other PR?

**Comment by zmb3** [comment]
@ptgott I finally got back to this. Should be a much simpler review now that the rest of your comments have already been addressed and merged. PTAL

File: docs/pages/desktop-access/getting-started.mdx
Comment (ptgott) [inline docs/pages/desktop-access/getting-started.mdx:156]:
```suggestion
These steps will need to be repeated if the Teleport's user certificate authority is rotated.
```

File: docs/pages/setup/operations/ca-rotation.mdx
Comment (ptgott) [inline docs/pages/setup/operations/ca-rotation.mdx:69]:
```suggestion
Teleport signs Windows Desktop services with the user certificate authority.
```

File: docs/pages/setup/operations/ca-rotation.mdx
Comment (ptgott) [inline docs/pages/setup/operations/ca-rotation.mdx:71]: Should there be an article/possessive adjective before "group policy"?

**Review by ptgott — APPROVED** [review]
Approved with small comments

File: docs/pages/desktop-access/getting-started.mdx
Comment (zmb3) [reply docs/pages/desktop-access/getting-started.mdx:156]: hmm. I don't think your suggestion is grammatically correct here.

Comment (ptgott) [reply docs/pages/desktop-access/getting-started.mdx:156]: Sorry, I'd meant to change that to "if the Teleport user's," but noticed that the original is fine as it is. Please ignore!

File: docs/pages/setup/operations/ca-rotation.mdx
Comment (zmb3) [reply docs/pages/setup/operations/ca-rotation.mdx:71]: No, I don't think so. "Group Policy" is standard terminology in Windows land.

Comment (ptgott) [reply docs/pages/setup/operations/ca-rotation.mdx:71]: 👍

**Review by r0mant — APPROVED** [review]

**Pull request merged**
